### ISSUE TITLE
create_link failing for device ios when "next_available_interface" is Loopback 0

### ISSUE THREAD

**Issue by gmuloc** [CiscoDevNet/virl2-client#3]
When using the method `connect_two_nodes` for two ios nodes
https://github.com/CiscoDevNet/virl2-client/blob/master/virl2_client/models/lab.py#L581

I get an Exception raised.

This occur when the `next_available_interface` is Loopback0 - I think the JSON payload returned here https://github.com/CiscoDevNet/virl2-client/blob/master/virl2_client/models/lab.py#L571 does not contain the key `id` most certainly because it makes no sense to create a link betwween Loopback0 and something else so probably it should be caught here.

Debugging logs below:

```
15:20:35,760 INFO: *** Adding link between Node: ios-1 and Node: ios-2
15:20:35,760 WARNING: Interface: Loopback0 <<<<<<<<<<< this is the output of node1.next_available_interface() or node1.create_interface()
15:20:35,806 WARNING: Interface: GigabitEthernet0/0
15:20:35,847 ERROR: API Error: {
  "description": "The API at /api/v0/labs/fd2fab/links encountered an unexpected error.  Please report this problem to support.",
  "code": 500
}
Traceback (most recent call last):
  File "virl2.py", line 125, in <module>
    add_n_devices(lab, "ios-", "iosv", n)
  File "virl2.py", line 111, in add_n_devices
    create_link(lab, previous_device, device)
  File "virl2.py", line 68, in create_link
    lab.connect_two_nodes(d1, d2)
  File "/Users/gmuloche/.pyenv/versions/3.7.7/lib/python3.7/site-packages/virl2_client/models/lab.py", line 594, in connect_two_nodes
    return self.create_link(iface1, iface2)
  File "/Users/gmuloche/.pyenv/versions/3.7.7/lib/python3.7/site-packages/virl2_client/models/lab.py", line 573, in create_link
    link_id = result["id"]
KeyError: 'id'
```

**Comment by rschmied** [comment]
thanks for looking into this... I need to fix this into the BitBucket code repository as a start.

**Comment by rschmied** [comment]
i've fixed this, will be part of the 2.1 code drop.
